SimPEG/utils/model_utils.py

```diff
@@ -260,3 +260,73 @@ def ismember(a, b):
     actv[inds] = True
 
     return actv
+
+
+    def depth_weighting(mesh, topo, indActive, v, z0):
+    """
+    Create depth weighting
+
+    Parameters
+    ----------
+    mesh : Mesh object
+        discretize model space.
+    topo : numpy.ndarray
+        [X,Y,Z] topographic data.
+    indActive : array of bool
+        index vector for the active cells on the mesh below the topography
+    v : float
+        exponent constant parameter for depth weighting.
```

Comment:
probably a good idea to supply a default

Reply:
I agree, will do that!